src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -753,6 +800,8 @@ public static class CurrentProcessingFileInformation {
         private long startPosition;
         @JsonProperty(PERSISTED_CURRENT_PROCESSING_FILE_LAST_MODIFIED_TIME)
         private long lastModifiedTime;
+        @JsonProperty(PERSISTED_CURRENT_PROCESSING_FILE_LAST_ACCESSED_TIME)
```

Comment:
are we even using Jackson on this object?

If we are, make sure that the jackson we use is configured to allow for unknown properties otherwise this will break if someone downgrades.

Reply:
Great point I wasn't too familiar with Jackson, but it seems this might be leftovers from some legacy code. Might be able to remove them